mariner-installer/build/build

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -ex
+
+echo "BRANCH: $BRANCH"
+cd $WORKSPACE/mariner-installer
+
+. /etc/os-release
```

Comment:
My opinion on why this particular way of detecting the host info shouldn't be used:

    [vagrant@localhost ~]$ cat /etc/os-release
    cat: /etc/os-release: No such file or directory
    [vagrant@localhost ~]$ cat /etc/redhat-release
    CentOS release 6.5 (Final)

I understand this is for Ubuntu/Debian only, but there are other ways that are cross-platform and would help normalize what we use across ceph-build already